nimare/tests/utils.py

```diff
@@ -120,3 +123,33 @@ def _transform_res(meta, meta_res, corr):
     if isinstance(corr_expectation, type(pytest.raises(ValueError))):
         pytest.xfail("this meta-analysis & corrector combo fails")
     return cres
+
+
+def standardize_field(dataset, metadata):
+    """Document This."""
+    # moderators = dataset.annotations[metadata]
+    categorical_metadata, numerical_metadata = [], []
+    for metadata_name in metadata:
+        if np.array_equal(
+            dataset.annotations[metadata_name], dataset.annotations[metadata_name].astype(str)
+        ):
+            categorical_metadata.append(metadata_name)
+        elif np.array_equal(
+            dataset.annotations[metadata_name], dataset.annotations[metadata_name].astype(float)
+        ):
+            numerical_metadata.append(metadata_name)
+    if len(categorical_metadata) > 0:
+        LGR.warning(f"Categorical metadata {categorical_metadata} can't be standardized.")
+    if len(numerical_metadata) == 0:
+        raise ValueError("No numerical metadata found.")
+
+    moderators = dataset.annotations[numerical_metadata]
+    standardize_moderators = moderators - np.mean(moderators, axis=0)
+    standardize_moderators /= np.std(standardize_moderators, axis=0)
```

Comment:
this function can be deleted, it's functionality is replicated in the `StandardizeField` class

Reply:
Deleted :)